src/modules/jcms_notifications/drush/jcms_notifications.drush.inc

```diff
@@ -160,7 +160,7 @@ function drush_jcms_notifications_send_notifications() {
     foreach ($nodes as $node) {
       $nids[] = $node->id();
       $sns_crud->sendMessage($node);
```

Comment:
The `sendMessage` method returns a `BusOutgoingMessage` object. You can get the ID from this.
```
$message = $sns_crud->sendMessage($node);
$id = $message->getId();
```